manifests/base/redis/redis.yaml

```
@@ -0,0 +1,59 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: redis
+spec:
+  serviceName: redis
+  replicas: 1
+  selector:
+    matchLabels: {}
+  template:
+    metadata: {}
+    spec:
+      containers:
+      - name: redis
+        image: "redis:6.0.20"
+        ports:
+        - containerPort: 6379
+        env:
+          - name: REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: redis-password
+                name: redis-secrets
+        volumeMounts:
+        - name: redis-data
+          mountPath: /data
+  volumeClaimTemplates:
+  - metadata:
+      name: redis-data
+    spec:
+      accessModes:
+      - ReadWriteOnce
```

ReadWriteOnce feels weird, you sure about it?
```suggestion
      accessModes:
      - ReadWriteMany
```

---

Yeah it is a read write once

---

Confirmed this morning.